webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/SystemProperties/index.js

```diff
@@ -0,0 +1,219 @@
+      }}
+    >
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Model')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {model}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Host group')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {hostgroupName}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Owner')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {ownerName}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Owner type')}</DescriptionListTerm>
+        <DescriptionListDescription>{ownerType}</DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Location')}</DescriptionListTerm>
+        <DescriptionListDescription>{location}</DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Organization')}</DescriptionListTerm>
+        <DescriptionListDescription>{organization}</DescriptionListDescription>
+      </DescriptionListGroup>
+      <Slot id="host-details-tab-properties-2" multi />
+    </DescriptionList>
+    <Divider className="padded-divider" />
+    <DescriptionList isCompact>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Created at')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {createdAt && <LongDateTime date={createdAt} />}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Updated at')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {updateAt && <LongDateTime date={updateAt} />}
+          </SkeletonLoader>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Uptime (s)')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <SkeletonLoader
+            status={status}
+            emptyState={<DefaultLoaderEmptyState />}
+          >
+            {uptime && <RelativeDateTime date={formatBootTime(uptime)} />}
```

Comment:
Why do you insist on passing the uptime? We have a normalized entry for the boot time available. That's much better because in API it could be cached. With an uptime you're guaranteed to always invalidate your cache.

If the boot time is somehow not available in the content, we should fix that instead of normalizing it back to a boot time. The database has stored it as a static boot time. We shouldn't be converting it back to an uptime and then back to a boot time.

Reply:
Maybe I miss here something, but the `boot_time` attribute is missing from the host API rabl, and adding it to the attributes list ends with nothing 🤔

Reply:
It's a facet. I do see there's some code for the facet definitions:
https://github.com/theforeman/foreman/blob/d6c44dcf85424da80f8d9d4de5bf9c0f57b19a10/app/views/api/v2/hosts/show.json.rabl#L9-L14

However, I don't know for sure how we would get this to show up. @ezr-ondrej any thoughts?

On an unrelated note: there's quite a bit of duplication in our API. `puppet_proxy_id`, `puppet_proxy_name` and `puppet_proxy` where the latter is a hash with the ID, name and URL of the Puppet proxy. Same for the Puppet CA Proxy and monitoring proxy.


@ezr-ondrej

Reply:
Let me get into this in separate PR, I think we've not included the facet attributes in API in the original PR, so we should fix that.

I won't comment on the API output ^_^ I believe it is long overdue to have documentation for API outputs in some standartized format (swagger?) that would "force" us to standartize how we deal with relations in API outputs. FWIW this duplication copies the input parameters where you can input id/name or something in between for the resource. I do not know enough to dig into this and it's definitelly faar from scope of this PR ^_^

Reply:
Actually we have `uptime_seconds` from ac95b80e2e9281c44b8cb31484255d83853d5676 it is not great, it should have been handled in true facet approach, but it is there ^_^

Reply:
#9089 for doing this properly, but @amirfefer please feel free to use `uptime_seconds` ^_^

Reply:
A bit off topic for this PR, but I'll use it to rant. I also believe we should have strict descriptions of the API output. Right now https://apidocs.theforeman.org/ is rather hard to use. You have to sort of guess the fields from some examples. Recently a user was also complaining about this and I think it was a completely fair complaint.

Reply:
Yup, I definitelly agree, it is terribe user experience when you just have to get some data and see what you get.